Examples/CaseStudies/SwiftUICaseStudies/04-HigherOrderReducers-Lifecycle.swift

```diff
@@ -19,8 +19,8 @@ struct LifecycleReducer<Wrapped: ReducerProtocol>: ReducerProtocol {
   }
 
   let wrapped: Wrapped
-  let onAppear: Effect<Wrapped.Action, Never>
-  let onDisappear: Effect<Never, Never>
+  let onAppear: EffectTask<Wrapped.Action>
+  let onDisappear: EffectTask<Never>
```

Comment:
Haha, this is wonky, but so is this case study, which will probably be removed when we ship our navigation APIs.